packages/vite/src/node/plugins/importAnalysisBuild.ts

```diff
@@ -483,6 +485,16 @@ export function buildImportAnalysisPlugin(config: ResolvedConfig): Plugin {
             }
           }
 
+          s.append(`\
+function __viteFile(indexes) {
+  const deps = ${JSON.stringify(
```

Comment:
Shouldn't we move this file out to avoid memory relocation on each call?

Reply:
I was thinking that JS engines should optimize this for us, though I've not measured it.

Reply:
Renamed to `__viteMapDep`.